charts/ark-cloudconfig/templates/storageclass.yaml

```diff
@@ -0,0 +1,11 @@
+kind: StorageClass
```

Comment:
I think a Helm chart should not define a StorageClass. A StorageClass depends on the underlying technology of the Kubernetes cluster. Here, it seems to me that it won't be possible to deploy this Helm chart on minikube on developer's laptops...

Reply:
Currently it does not support minikube or other cloud providers. CloudConfig server requires network file storage that is shared with other arkcase applications. Will work on making CloudConfig to work on Minikube and other cloud providers in  later sprints.

Reply:
OK